server.go

```diff
@@ -4287,8 +4287,36 @@ func (s *server) fetchLastChanUpdate() func(lnwire.ShortChannelID) (
 
 // applyChannelUpdate applies the channel update to the different sub-systems of
 // the server.
-func (s *server) applyChannelUpdate(update *lnwire.ChannelUpdate) error {
+func (s *server) applyChannelUpdate(update *lnwire.ChannelUpdate, outpoint wire.OutPoint,
+	disabled bool) error {
+
+	// Apply the channel update to the outgoing links in the switch. This
+	// helps keep the switch in sync with certain policy actions like
+	// manual disabling of channel.
+	//
+	// NOTE: disabling a channel only disables outgoing payments, not incoming.
+	var edgePolicy *channeldb.ChannelEdgePolicy
+	_, edgePolicy, edgePolicy2, err := s.graphDB.FetchChannelEdgesByOutpoint(&outpoint)
+	if err != nil {
+		return err
+	}
+	if edgePolicy.Node.PubKeyBytes == s.currentNodeAnn.NodeID {
```

Comment:
Fairly certain it's possible for one of `edgePolicy` or `edgePolicy2` to be nil here.
Also I believe this should be:
```diff
--- if edgePolicy.Node.PubKeyBytes == s.currentNodeAnn.NodeID
+++ if edgePolicy.Node.PubKeyBytes != s.currentNodeAnn.NodeID
```